plaid/accounts.go

```diff
@@ -0,0 +1,100 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type Account struct {
+	ID      string `json:"account_id"`
+	Balance struct {
+		Available float64 `json:"available"`
+		Current   float64 `json:"current"`
+		Limit     float64 `json:"limit"`
```

Comment:
Balance also has an `iso_currency_code` and `unofficial_currency_code` field for the new api version.

Reply:
updated!